lib/app/features/chat/presentation/pages/chat_people_card.dart

```diff
@@ -55,6 +74,62 @@ class ChatPeopleCard extends StatelessWidget {
       ),
     );
   }
+
+  Widget _buildBadgeWidget(List<UserDetailBadgeEntity>? badges) {
+    if (badges == null) {
+      return const SizedBox.shrink();
+    } else {
+      var onlyInlineBadges = <UserDetailBadgeEntity>[];
+
+      for (final badge in badges) {
+        if (badge.style != 'inline-block') {
+          onlyInlineBadges.add(badge);
+        }
+      }
+
+      return Row(
+        children: onlyInlineBadges
+            .map((badge) => Padding(
+                padding: const EdgeInsets.only(left: 4.0),
+                child: UserBadgeWidget(
+                  badgeDescription: badge.description,
+                  badgeImageUrl: badge.imageUrl,
+                  badgeName: badge.name,
+                  badgePopUp: badge.popUp,
+                  badgeShowDescription: badge.showDescription,
+                )))
+            .toList(),
+      );
+    }
+  }
+}
+
+Widget _buildCloseUser(List<UserDetailBadgeEntity>? badges) {
+  if (badges == null) {
+    return const SizedBox.shrink();
+  } else {
+    var _emptyBadge = UserDetailBadgeEntity(
+        code: '',
+        description: '',
+        imageUrl: '',
+        name: '',
+        popUp: 0,
+        showDescription: 0,
+        style: '');
+    final badge = badges.firstWhere(
+      (badge) => badge.style == 'inline-block',
+      orElse: () => _emptyBadge,
+    );
+    if (badge.style != '') {
+      return UserCloseBadgeWidget(
+        badgeImageUrl: badge.imageUrl,
+        badgeName: badge.name,
+        badgePopUp: badge.popUp,
+      );
+    } else {
+      return const SizedBox.shrink();
+    }
```

Comment:
Ler `if` é um pouco complexo, e por isto é importante pensar em maneiras que permita uma leitura mais fluida. Não estou dizendo que o código é ruim, só estou dizendo que é mais complexo ficar fazendo mentalmente validações booleano para ler o código, diminui o rítmico do próximo desenvolvedor ler o código. A minha sugestão.

```suggestion
    if (badge.style.isEmpty) {
        return const SizedBox.shrink();
    }
    
      return UserCloseBadgeWidget(
        badgeImageUrl: badge.imageUrl,
        badgeName: badge.name,
        badgePopUp: badge.popUp,
      );
```

Neste código eu simplifico a condição, se acontecer esta condição o código para, se não ocorrer continua o fluxo.